Modules/booking/Model/BookingInvoice.php

```diff
@@ -152,7 +152,21 @@ public function invoice($id_space, $beginPeriod, $endPeriod, $id_resp, $invoice_
             $reservations = $modelCal->getUnpricedReservations($id_space, $beginPeriod, $endPeriod, $res["id"], $id_resp);
 
             // get list of quantities
-            $calQuantities = $bkCalQuantitiesModel->getByResource($id_space, $res["id"]);
+            $allCalQuantities = $bkCalQuantitiesModel->getByResource($id_space, $res["id"], include_deleted:true);
+
+            // Get non-deleted quantities
+            $calQuantities = array_filter($allCalQuantities, function($calQte) {
+                return $calQte["deleted"] == 0;
+            });
+
+            // Get deleted quantities which are not used as invoicing unit
```

Comment:
"which are used" , no ? you're testing  $calQte["is_invoicing_unit"] ==

Reply:
May be a github copilot joke :/ ...